docs/poc2023q1/outline.md

```diff
@@ -0,0 +1,257 @@
+
+See [the overview picture](Edge-PoC-2023q1.svg) for an overview
+picture.
+
+## Inventory Management workspaces
+
+This design takes as a given that something maintains some kcp
+workspaces that contain dynamic collections of Location and SyncTarget
+objects as defined in [kcp
+TMC](https://github.com/kcp-dev/kcp/tree/main/pkg/apis), and that one
+view can be used to read those Location objects and one view can be
+used to read those SyncTarget objects.
+
+To enable the workload management layer to manipulate each edge
+cluster, that cluster's SyncTarget has an annotation that refers to a
+Secret that holds a kubeconfig that can be used by the Bootstrapper to
+install the edge-mc machinery in the edge cluster.  These annotations
+and Secrets are created by the inventory management layer for use by
+the workload management layer.
+
+To complete the plumbing of the syncers, for each SyncTarget the
+mailbox controller creates the following needed accompanying items in
+that SyncTarget's workspace (which must authorize the mailbox
+controller to do this).
+
+1. A ServiceAccount that the syncer will authenticate as.
+2. A ClusterRole manipulating that SyncTarget and the
+   APIResourceImports (what are these?).
+3. A ClusterRoleBinding that links that ServiceAccount with that
+   ClusterRole.
+
+FYI, those are the things that `kubectl kcp workload sync` directly
+creates besides the SyncTarget.
+
+## Edge Service Provider workspace
+
+The edge multi-cluster service is provided by one workspace that
+includes the following things.
+
+- An APIExport of the edge API group.
+- The edge controllers: scheduler, placement translator, bootstrapper,
+  mailbox controller, and status sumarizer.
+
+## Workload Management workspaces
+
+The users of edge multi-cluster primarily maintain these.  Each one
+contains the following things.
+
+- APIBindings to the APIExports of workload object types.
+- A workload spec.  That is, some namespaces containing namespaced
+  workload objects --- and possibly later on some non-namespaced
+  workload objects.  Initially the workload objects must be of types
+  drawn from a list of those supported by the EMC implementation
+  (because extension by user CRDs is not supported yet).
+- EMC interface objects: EdgePlacement, SinglePlacementSlice, those
+  for customization and summarization.
+
+## Mailbox workspaces
+
+The mailbox controller maintains one mailbox workspace for each
+SyncTarget.  A mailbox workspace acts as a workload source for TMC,
+prescribing the workload to go to the corresponding edge pcluster and
+holding the corresponding TMC Placement object.
+
+A mailbox workspace contains the following items.
+
+1. APIBindings (maintained by the mailbox controller) to APIExports of
+   workload object types.
+2. Workload namespaces holding workload objects, post customization.
+3. A TMC Placement object.
+
+## Edge cluster
+
+Also called edge pcluster.
+
+One of these contains the following items, maintained by the
```

Comment:
maintained by the bootstrapper or only provisioned by the bootstrapper?  Maybe we can maintain/manage them through KCP once they are provisioned?